lib/node-utils.ts

```diff
@@ -253,3 +253,22 @@ export function getImportModuleName(
     return node.arguments[0].value;
   }
 }
+
+export function getSpecifierFromImport(
+  node: ImportModuleNode,
+  specifierName: string
+) {
+  if (isImportDeclaration(node)) {
+    const namedExport = node.specifiers.find(
+      (node) => isImportSpecifier(node) && node.imported.name === specifierName
+    );
+    // it is "import { foo } from 'baz'""
```

Comment:
What about `import { foo as qux } from 'bar'`?

Reply:
it is covered (the tests cover this scenario), but the comment is not reflecting it - I will update it

Reply:
done